src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -371,23 +372,31 @@ public function filter_add_body_classes( $classes ) {
 		$setting  = $this->get_template_setting();
 		$template = $this->get_template_object()->get_path();
 
-		if ( 'page' === $setting ) {
-			$classes[] = 'page-template-' . sanitize_title( $template );
+		if ( 'page' !== $setting ) {
+			return $classes;
+		}
+
+		$classes[] = 'page-template-' . sanitize_title( $template );
 
-			if ( ! is_tax() ) {
-				$key = array_search( 'archive', $classes );
+		if ( ! is_tax() ) {
+			$key = array_search( 'archive', $classes );
 
-				if ( false !== $key ) {
-					unset( $classes[ $key ] );
-				}
+			if ( false !== $key ) {
+				unset( $classes[ $key ] );
 			}
-		} else {
-			$classes[] = 'tribe-events-page-template';
 		}
 
 		return $classes;
 	}
 
+	public function add_body_classes() {
+		$setting  = $this->get_template_setting();
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 9365f10dc3830d7965585be61e2fec21</sub>